### DISCUSSION TITLE
To be able to edit Report Design in TE3

### DISCUSSION THREAD

**Discussion by databayes** [TabularEditor/TabularEditor3#1309] [Ideas]
### Is your feature request related to a problem? Please describe

With the ability to customise the visual layer, there appears to be many more buttons to click and click and click.  Very time consuming.  Is there scope / plan to be able to use TE to enable to speed up Report Design in the same way it helps with modelling and building?

### Describe the solution you'd like

To speed up report design within TE3 so that I can use one tool for report building and visualisation

### Goals

1. Improve time to build visual layer
2.

**Comment by otykier** [comment]
Hi Susan,
This is certainly an interesting idea, and especially now that Microsoft is working on allowing external tools to access and modify the visual layer of a report. However, I'm afraid that it's too far outside of our scope. Tabular Editor (and especially TE3), already has a very large amount of features, to the point where it gets confusing or overwhelming to new users. Adding an entirely new design surface would make this problem worse, and moreover, the core competencies within the team, is centered around the semantic model. Also, this is no small investment. Believe it or not, but the visual metadata is substantially more complex than the tabular object metadata :)
We'll be more than happy to support any community effort to build such a tool, but we probably won't be the ones in charge.

**Comment by databayes** [comment]
Thanks for replying Daniel, and I can only imagine the complexity involved with the visual layer and yes, agree new Users.  Love the new core visuals but so many clicks.

Am speaking at East Midlands User group re TE on Tuesday and Data Ceili in Ireland on Friday re TE.  Those sessions are both on TE3 which I think will be good for East Midlands as it's Paul Andrew's User Group so probably fairly advanced PBI users.  https://www.meetup.com/eastmidlandsdata/events/297376332/

Data Ceili picked the TE3 session but it's a free community session so they might prefer the intro to TE2 but can check with them when I get there.

Have been asked to speak at Ken Puls User Group in Canada as well in July and have offered to do 30 mins on TE2 with a short intro to TE3 as I think his user base is primarily Excel users and some BI.

Hope you are well and glad you liked the other ideas as well,
Sue 🙂

**Reply by otykier** [reply]
Thank you for your incredible dedication and enthusiasm in promoting Tabular Editor. Your efforts in educating and engaging with the community are immensely valuable to us.

Your advocacy and expertise not only help others get the most out of Tabular Editor but also contribute significantly to the growth and strength of our user community. We are incredibly grateful for your support and the positive impact you make.

Thank you again for all that you do, and best of luck with your upcoming sessions! Let us know if we can be of any help!

**Comment by databayes** [comment]
It's no problem, having the Enterprise edition meant I could attach to a semantic model in the service and share screens with  my customer, a job on a horribly complex model took an hour rather than hours.  TE helps me no end and I like conferences 🙂 🙂
I do have passing the course and the training course on my to do list by end of the year,
Sue
